orchagent/portsorch.cpp

```diff
@@ -1860,6 +1886,24 @@ void PortsOrch::initPortSupportedSpeeds(const std::string& alias, sai_object_id_
     m_portStateTable.set(alias, v);
 }
 
+void PortsOrch::initPortCapAutoNeg(Port &port)
+{
+    sai_status_t status;
+    sai_attribute_t attr;
+
+    attr.id = SAI_PORT_ATTR_SUPPORTED_AUTO_NEG_MODE;
```

Comment:
Yes, it will cause breakage, hence the AN capability will be updated to use 1(supported) by default

Reply:
Sure. Please use this capability to a warning log in case if SAI returns not supported for the attribute. SAI vendors can then mitigate to implement this attribute without the functional impact.

Reply:
Done